src/utils/api.js

```diff
@@ -32,3 +32,9 @@ export function postComment(articleId, username, body) {
 export function deleteComment(commentId) {
   return ncNewsAPI.delete(`/comments/${commentId}`);
 }
+
+export function getAllTopics() {
+  return ncNewsAPI.get("/topics");
+}
```

Comment:
Nice to see you are using the api to filter topics instead of fetching all and locally filtering 👍